README.adoc

```diff
@@ -476,6 +476,15 @@ This means, for example, that using polling on file mounted secret sources does
 **Notes**:
 - Properties under *spring.cloud.kubernetes.reload.* should not be used in config maps or secrets: changing such properties at runtime may lead to unexpected results;
 - Deleting a property or the whole config map does not restore the original state of the beans when using the *refresh* level.
+- You need to have the reload actuator dependency in your pom file for reload and the reload actuator enabled `management.endpoint.restart.enabled=true` in your `bootstrap.yaml` or `application.yaml`:
+
+```xml
+<dependency>
```

Comment:
If this is needed we should specify the spring boot starter here

Reply:
We probably should include the dependency in `spring-cloud-starter-kubernetes-config`

Reply:
good catch, switched it across to use the starter